# This file contains the nominal poses of the each Aruco markers
# that is to be used in the test rig in Millimeters. Each aruco marker's 
# pose(mm)/orientation(rad) are to be measured and inserted as a 4x4 Homogeneous matrix

'testrig_aruco_marker_info':
  'base_left_marker_pose':
    - - 1.0
      - 0.0
      - 0.0
      - 0.0
    - - 0.0
      - 1.0
      - 0.0
      - 0.0
    - - 0.0
      - 0.0
      - 1.0
      - 0.0
    - - 0.0
      - 0.0
      - 0.0
      - 1.0    
  'base_right_marker_pose':
    - - 1.0
      - 0.0
      - 0.0
      - 0.0
    - - 0.0
      - 1.0
      - 0.0
      - 0.0
    - - 0.0
      - 0.0
      - 1.0
      - 0.0
    - - 0.0
      - 0.0
      - 0.0
      - 1.0

  'shoulder_marker_pose':
    - - 1.0
      - 0.0
      - 0.0
      - 0.0
    - - 0.0
      - 1.0
      - 0.0
      - 0.0
    - - 0.0
      - 0.0
      - 1.0
      - 0.0
    - - 0.0
      - 0.0
      - 0.0
      - 1.0

  'wrist_inside_marker_pose':
    - - 1.0
      - 0.0
      - 0.0
      - 0.0
    - - 0.0
      - 1.0
      - 0.0
      - 0.0
    - - 0.0
      - 0.0
      - 1.0
      - 0.0
    - - 0.0
      - 0.0
      - 0.0
      - 1.0

  'wrist_top_marker_pose':
    - - 1.0
      - 0.0
      - 0.0
      - 0.0
    - - 0.0
      - 1.0
      - 0.0
      - 0.0
    - - 0.0
      - 0.0
      - 1.0
      - 0.0
    - - 0.0
      - 0.0
      - 0.0
      - 1.0